notebooks/407-person-tracking-webcam/407-person-tracking.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<p>can you please provide link on article, where deep sort algo is described here too? and also for other terminology which is behind of scene (Kalman filter, Mahalanobis distance, Hungarian algorithm e.t.c.)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/751/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>